content/wasm-languages/zig.md

```diff
@@ -27,29 +27,103 @@ For the most part, write your Zig code as usual. When compiling, use the target
 
 Things we like:
 
-- The Zig compiler is so nice, we now use it to compile our C as well as our Zig.
+- The Zig compiler is so nice that we now use it to compile our C and Zig.
 
 Things we're not big fans of:
 
-- When we get Wasm-related errors, they can be really terse.
+- When we get Wasm-related errors, they can be really terse. However, since the original publishing of this page (around Zig 0.4.0), updates have been made to the error section of the official Zig documentation e.g. explaining how [Zig's error handling](https://ziglang.org/documentation/0.11.0/#toc-Common-errdefer-Slip-Ups) includes `defer` statements and `errdefer`, which triggers on block-exit errors. For example:
 
-## Example
+```
+fn createFoo(param: i32) !Foo {
+    const foo = try tryToAllocateFoo();
+    // now we have allocated foo. we need to free it if the function fails.
+    // but we want to return it if the function succeeds.
+    errdefer deallocateFoo(foo);
+
+    const tmp_buf = allocateTmpBuffer() orelse return error.OutOfMemory;
+    // tmp_buf is truly a temporary resource, and we for sure want to clean it up
+    // before this block leaves scope
+    defer deallocateTmpBuffer(tmp_buf);
+
+    if (param > 1337) return error.InvalidParam;
+
+    // here the errdefer will not run since we're returning success from the function.
+    // but the defer will run!
+    return foo;
+}
+```
+
+# Examples
 
->> All of our examples follow [a documented pattern using common tools](/wasm-languages/about-examples).
+>> All of our examples follow [a documented pattern using common tools](/wasm-languages/about-examples). In these examples, we initially run our applications using the [wasmtime](https://github.com/bytecodealliance/wasmtime) runtime and then progress and demonstrate how to run Zig applications using the [Spin framework](https://developer.fermyon.com/spin/v2/index).
```

Comment:
```suggestion
>> All of our examples follow [a documented pattern using common tools](/wasm-languages/about-examples). In these examples, we initially run our applications using the [wasmtime](https://github.com/bytecodealliance/wasmtime) runtime and then progress and demonstrate how to run Zig applications using the [Spin framework](/spin/v2/index).
```

Reply:
Ended up adding `../spin/v2/*` because we are one layer down here in `wasm-languages`.
Thanks for spotting this absolute / relative issue 💯